macos-redirector/MitmproxyAppleTunnel.app/Contents/Info.plist

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>BuildMachineOSBuild</key>
+	<string>22F82</string>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>MitmproxyAppleTunnel</string>
+	<key>CFBundleIconFile</key>
+	<string>AppIcon</string>
+	<key>CFBundleIconName</key>
+	<string>AppIcon</string>
+	<key>CFBundleIdentifier</key>
+	<string>com.emanuele.MitmproxyAppleTunnel</string>
```

Comment:
Could we eventually have this self-identify as `org.mitmproxy.MacOsRedirector` (or `org.mitmproxy.macos-redirector`)? :)

Reply:
of course!